index.js

```diff
@@ -141,6 +141,7 @@ class XiaomiRoborockVacuum {
       .on("get", (cb) => callbackify(() => this.getSerialNumber(), cb));
 
     this.services.fan = new Service.Fan(this.config.name, "Speed");
+    this.services.fan.setPrimaryService(true);
```

Comment:
I'm getting an error after update to 0.8.0
```
[5/11/2020, 8:46:49 PM] TypeError: this.services.fan.setPrimaryService is not a function
    at XiaomiRoborockVacuum.initialiseServices (/homebridge/node_modules/homebridge-xiaomi-roborock-vacuum/index.js:144:23)
    at new XiaomiRoborockVacuum (/homebridge/node_modules/homebridge-xiaomi-roborock-vacuum/index.js:120:10)
    at Server._loadAccessories (/usr/local/lib/node_modules/homebridge/lib/server.js:308:29)
    at Server.run (/usr/local/lib/node_modules/homebridge/lib/server.js:91:38)
    at module.exports (/usr/local/lib/node_modules/homebridge/lib/cli.js:59:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:1156:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
```

Reply:
Are you running the latest `homebridge`?

Reply:
I was running 0.4.53 - seems to be fine after updating to 1.0.4

Reply:
Brilliant! Thanks for confirming! I published `0.8.1` anyway to validate the function exists before calling it to keep backwards compatibility.